2_base_and_sensor.ipynb

```diff
@@ -172,40 +316,78 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "壁にぶつかってしまいました。上の■ボタンを押して動作を停止させましょう。\n",
     "![title](./imgs/2_stop_button.png)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Oops, the robot bumped into the wall. Press the ■ button above to stop our code.\n",
```

Comment:
"Oops!  The robot bumped into the wall. Press the ■ button above to stop our code.\n",